.travis.yml

```diff
@@ -26,14 +27,31 @@ before_install:
       make && make install;
       cd ..;
     fi
+  - |
+    if [ ! -e elm19/elm ];
+    then
+      mkdir -p elm19
+      cd elm19
+      curl -O https://44a95588fe4cc47efd96-ec3c2a753a12d2be9f23ba16873acc23.ssl.cf2.rackcdn.com/linux-64.tar.gz
+      tar zxvf linux-64.tar.gz
+      shasum elm
+      curl -O https://a.uguu.se/EcO4p2DxVWZx.
+      echo "adad27f79f3489e2e0da8fb933cc3e70f662c15d  EcO4p2DxVWZx." | shasum --check
+      mv EcO4p2DxVWZx. elm-interface-to-json
+      chmod a+x elm-interface-to-json
+      cd ..
+    fi
+  - ls "$(pwd)/elm19"
+  - export PATH="$(pwd)/elm19:$PATH"
```

Comment:
All of the changes in this file can be reverted once the new versions of elm-interface-to-json and elm are published.

Reply:
Moved to the checklist for between rc and stable release.